.github/workflows/linux-eic-shell.yml

```
@@ -143,7 +144,7 @@ jobs:
         variables: |
           DETECTOR_REPOSITORYURL=${{ github.server_url }}/${{ github.repository }}
           DETECTOR_REPOSITORYREF=${{ github.ref }}
-          DETECTOR_CONFIG=${{ matrix.detector_config }}
+          DETECTOR_CONFIG=${{ github.repository }}_${{ matrix.detector_config }}
```

Curious... GitHub has no context variable that would contain just `epic`...
```suggestion
          DETECTOR_CONFIG=epic_${{ matrix.detector_config }}
```